assignment4/src/app/game-controll/game-controll.component.ts

```diff
@@ -0,0 +1,43 @@
+import { OnDestroy } from '@angular/core';
+import {
+  Component,
+  OnInit,
+  Output,
+  EventEmitter,
+  } from '@angular/core';
+@Component({
+  selector: 'app-game-controll',
+  templateUrl: './game-controll.component.html',
+  styleUrls: ['./game-controll.component.css']
+})
+export class GameControllComponent implements
+OnInit, OnDestroy {
+  @Output() started = new EventEmitter<number>();
+  // tslint:disable-next-line: no-output-native
```

Comment:
Very bad coding practice to disable tslint messages. This should only be done in special cases. Please reconsider it.